src/components/pageNav/index.jsx

```diff
@@ -26,14 +26,15 @@ class PageNav extends React.PureComponent {
         icon
         className="outlined"
         to={buttonLink}
-        component={Link}
+        component={type === 'previous' ? Link : disableButton ? null : Link}
```

Comment:
so, with the above change, is always false for previous button so we can simplify this to `component={disableButton ? null : Link}`